autopush/main_argparse.py

```diff
@@ -103,6 +103,10 @@ def add_shared_args(parser):
                         help="Use the cryptography library vs. JOSE",
                         action="store_true",
                         default=False, env_var="USE_CRYPTOGRAPHY")
+    parser.add_argument('--enable_simplepush',
+                        help="Enable the deprecated Simplepush protocol",
+                        action="store_true", default=False,
```

Comment:
Well, I figured that we were soft-killing Simplepush and that we could optionally re-enable it if we needed to. Making this default true means we have to mindfully disable it instead. I'll admit that both are equivalent, but one is more aggressive.

But, sure, fine, ruin my hopes for an early death for simplepush... ;)